modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/beefman.dm

```diff
@@ -0,0 +1,415 @@
+
+			// Pry it off...
+			user.visible_message("[user] grabs onto [p_their()] own [affecting.name] and pulls.", span_notice("You grab hold of your [affecting.name] and yank hard."))
+			if (!do_mob(user,target))
+				return TRUE
+
+			user.visible_message("[user]'s [affecting.name] comes right off in their hand.", span_notice("Your [affecting.name] pops right off."))
+			playsound(get_turf(user), 'sound/effects/meatslap.ogg', 40, 1)
+
+			// Destroy Limb, Drop Meat, Pick Up
+			var/obj/item/I = affecting.drop_limb() //  <--- This will return a meat vis drop_meat(), even if only Beefman limbs return anything. If this was another species' limb, it just comes off.
+			if (istype(I, /obj/item/food/meat/slab))
+				user.put_in_hands(I)
+
+			return TRUE
+	return ..()
+
+/datum/species/beefman/spec_unarmedattacked(mob/living/carbon/human/user, mob/living/carbon/human/target)
+	// Bleed On
+	if (user != target && user.is_bleeding())
+		target.add_mob_blood(user) //  from atoms.dm, this is how you bloody something!s
+	return ..()
+
+/datum/species/beefman/proc/handle_limb_mashing()
+	SIGNAL_HANDLER
+
+/datum/species/beefman/spec_attacked_by(obj/item/I, mob/living/user, obj/item/bodypart/affecting, mob/living/carbon/human/H, modifiers)
+	handle_limb_mashing()
+	// MEAT LIMBS: If our limb is missing, and we're using meat, stick it in!
+	if(LAZYACCESS(modifiers, RIGHT_CLICK))
+		return
+	if(H.stat < DEAD && !affecting && istype(I, /obj/item/food/meat/slab))
+		var/target_zone = user.zone_selected
+		var/list/limbs = list(BODY_ZONE_L_ARM, BODY_ZONE_R_ARM, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG)
+
+		if((target_zone in limbs))
+			if(user == H)
+				user.visible_message("[user] begins mashing [I] into [H]'s torso.", span_notice("You begin mashing [I] into your torso."))
+			else
+				user.visible_message("[user] begins mashing [I] into [H]'s torso.", span_notice("You begin mashing [I] into [H]'s torso."))
+
+			// Leave Melee Chain (so deleting the meat doesn't throw an error) <--- aka, deleting the meat that called this very proc.
+			spawn(1)
+				if(do_mob(user,H))
+					// Attach the part!
+					var/obj/item/bodypart/newBP = H.newBodyPart(target_zone, FALSE)
+					H.visible_message("The meat sprouts digits and becomes [H]'s new [newBP.name]!", span_notice("The meat sprouts digits and becomes your new [newBP.name]!"))
+					newBP.attach_limb(H)
+					if(I.reagents && I.reagents.total_volume)
+						I.reagents.trans_to(user, I.reagents.total_volume)	//transfer reagents to player
+					qdel(I)
+					playsound(get_turf(H), 'sound/effects/meatslap.ogg', 50, 1)
+
+			return TRUE // True CANCELS the sequence.
+
+	return ..() // TRUE FALSE
+
+/datum/species/beefman/after_equip_job(datum/job/J, mob/living/carbon/human/H, visualsOnly = FALSE)
+	to_chat(H, "<span class='danger'>Just because meatmen are from Fulpstation, which is more loose rule-wise than Skyrat, DOES NOT MEAN YOU ARE MAGICALLY ALLOWED TO BE LRP. ROLEPLAYING STANDARDS STILL APPLY.</span>")
```

Comment:
```suggestion
	to_chat(H, "<span class='danger'>Meatmen come from Fulpstation, a server with a looser ruleset than ours. This is NOT a pass to grief. Policy still applies fully!</span>")
```
I think this wording would be more appropiate

Reply:
Also why isn't it using the span macros?